src/main/java/seedu/address/logic/commands/ViewDriverTaskCommand.java

```diff
@@ -8,41 +8,43 @@
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.person.Driver;
-import seedu.address.model.task.Task;
 
 /**
- * List delivered task for specified driver
+ * List completed delivery task allocated to the particular driver.
  */
-public class ViewDriverTasksCommand extends Command {
+public class ViewDriverTaskCommand extends Command {
 
     public static final String COMMAND_WORD = "viewD";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Views the delivered tasks of the specified driver "
-            + "and displays them as a list with index numbers.\n"
-            + "Parameters: INDEX (must be a positive integer)"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "and displays them as a list with index numbers."
+            + "\n"
+            + "Parameters: " + "DRIVER_ID (must be a positive integer)"
+            + "\n"
+            + "Example: " + COMMAND_WORD + " "
+            + "1";
 
-    public static final String MESSAGE_SUCCESS = "listed delivered tasks";
+    public static final String MESSAGE_SUCCESS = "Listed %s completed tasks assigned to the Driver ID #%s";
 
     private int driverId;
 
-    public ViewDriverTasksCommand(int driverId) {
+    public ViewDriverTaskCommand(int driverId) {
         this.driverId = driverId;
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
+
         List<Driver> currentDriverList = model.getFilteredDriverList();
-        List<Task> taskList = model.getFilteredTaskList();
 
         if (driverId > currentDriverList.size()) {
             throw new CommandException(Messages.MESSAGE_INVALID_DRIVER_DISPLAYED_INDEX);
         }
```

Comment:
instead of checking the size of the driver list to see if the driver is present, a higher level way is to use model.hasDriver(driver id)

Reply:
Thx for the suggestion!